### ISSUE TITLE
Inform Users if any DNS Record has changed (so that External DNS are informed)

### ISSUE THREAD

**Issue by aspdye** [mail-in-a-box/mailinabox#649]

**Comment by JoshData** [comment]
You can use `management/dns_update.py --dump` or reverse-engineer the control panel's API to get the records, and then do your own diff.

**Comment by bronson** [comment]
Personally, I'd rather the system status checks ensure that each individual record is set correctly and email you if not.  Seems simpler and easier to use (?)

**Comment by aspdye** [comment]
Yes. We should definitly take a look about all the Records and not (only) about using the Nameservers.

**Comment by JoshData** [comment]
The status checks are already checking a handful of records, so checking more (SPF etc.) sounds good to me.

**Comment by bronson** [comment]
Currently the status page checks all A and MX records.  It doesn't check:

- SPF and DMARC for every domain name, including nameservers
- DKIM, caldav and carddav for every domain receiving mail

Once those are added, I think the status page will be checking all DNS configuration.  Can't promise anything but I'll attempt it next.

**Comment by bronson** [comment]
Also TXT records set by the user.

**Comment by bronson** [comment]
OK, I think this fix ready to go.  #955, #956, #957, and #958 are independent fixes that work together to solve this issue.

Tell me if there's anything I can correct!  Would like to use these on my domains.
